docker-compose.yml

```diff
@@ -0,0 +1,46 @@
+services:
+  db:
+    container_name: postgres_container
+    image: postgres
+    restart: always
+    ports:
+      - "5432:5432"
+    environment:
+      POSTGRES_USER: app_user
+      POSTGRES_PASSWORD: app_password
+      POSTGRES_DB: cyber_mysticism
```

Comment:
**Suggestion:** Consider using environment variables for sensitive data like `POSTGRES_USER`, `POSTGRES_PASSWORD`, and `POSTGRES_DB` instead of hardcoding them in the docker-compose file. This will increase the security of your application. [security]
```suggestion
      environment:
        POSTGRES_USER: ${POSTGRES_USER}
        POSTGRES_PASSWORD: ${POSTGRES_PASSWORD}
        POSTGRES_DB: ${POSTGRES_DB}
```